### DISCUSSION TITLE
Documentation on Operational Levels

### DISCUSSION THREAD

**Discussion by gurkerl83** [chakra-ui/panda#2624] [Ideas]
### Subject

Comparisons with Other Frameworks

### Description

It would be helpful for me and others if the documentation included a section that explains the operational levels of Panda CSS and other frameworks at first glance. The discussion at [this link](https://github.com/chakra-ui/panda/discussions/2568#discussioncomment-9338287) mentions, "we're merely offering a way to use Panda CSS through PostCSS," which I understand implies a feature of static extraction.

Could you provide more comprehensive documentation on this topic, particularly comparing it to other approaches, such as those pursued by Pigment CSS? [Reference link](https://github.com/mui/pigment-css/issues/6#issuecomment-2054102863)

Additionally, could you clarify if a JavaScript runtime is required to re-integrate the extracted CSS back into JSX when using Panda CSS?

Thanks!

**Comment by oliviertassinari** [comment]
> Could you provide more comprehensive documentation on this topic, particularly comparing it to other approaches, such as those pursued by Pigment CSS? https://github.com/mui/pigment-css/issues/6#issuecomment-2054102863

@gurkerl83 I have tried to assess the difference under https://www.notion.so/mui-org/pigment-css-Why-a-standalone-project-25205c31c8564e0daee6f79523de9b5c?pvs=4#95606c565c844f388385905f5379cc5e. I have worked on this docs when we were trying to figure if we could use any of the existing styling libraries for the next major of Material UI. This was before investing in Pigment CSS, it doesn't make much sense for MUI to get distracted with this unless we see a big gap and we are desperate for someone to go after it.
